File: java-main/java-main/Array/Ex1/p3.java
/*WAP to take size of array from user and also take integer elements from user Print product of odd index only*/

import java.util.*;
class Demo{
        public static void main(String[]args){
                Scanner sc = new Scanner(System.in);
                System.out.println("Enter array size:");
                int size=sc.nextInt();
                int arr[]=new int[size];
                int mul=1;
                System.out.println("Enter array element:");
                for(int i=0;i<arr.length;i++){
                        arr[i]=sc.nextInt();
                        if(arr[i]%2==0){
                                mul=mul*arr[i];
                        }
                }
                System.out.println("product of odd element ="+ mul);
        }
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/3d/p3.c
#include<stdio.h>
void main(){

	int arr[2][3][3]={1,2,3,4,5,6,7,8,9,10,11,12};

	printf("Array element are :\n");

	for(int i=0;i<2;i++){

		printf("Plane:%d\n",i);

		for(int j=0;j<2;j++){
			for(int k=0;k<2;k++){

				printf("%d ",arr[i][j][k]);
			}
			printf("\n");
		}
	}
}

